common/djangoapps/student/models.py

```diff
@@ -1526,6 +1526,11 @@ def refundable(self):
         # If it is after the refundable cutoff date they should not be refunded.
         refund_cutoff_date = self.refund_cutoff_date()
         if refund_cutoff_date and datetime.now(UTC) > refund_cutoff_date:
+            log.info(
+                u"user [%s] cannot be refunded because his/her cutoff date [%s] has passed",
+                self.username,
```

Comment:
user "self.user.username" Im not sure we have self.username. "self.user.username" is more appropriate .